docker/docker/src/main/java/org/arquillian/cube/docker/impl/docker/DockerClientExecutor.java

```diff
@@ -782,6 +784,15 @@ private void configureBuildCommand(Map<String, Object> params, BuildImageCmd bui
         if (params.containsKey(DOCKERFILE_NAME)) {
             buildImageCmd.withDockerfile(new File((String) params.get(DOCKERFILE_NAME)));
         }
+        
+        AuthConfig buildAuthConfig = new AuthConfig()
```

Comment:
Should we add here an if to make this only set in case of user set a username and password?